.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Run Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+  release:
+    types:
+      - created
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run tests
+        run: npm test
```

Comment:
add code coverage upload (not sure if your mocha invocation generates coverage files - if it doesn't, it should)
See https://github.com/adobe/aio-cli-plugin-app/blob/1f35c492776a360dc30c102498b3fc997f6885f2/.github/workflows/node.js.yml#L32-L36

Reply:
(code coverage for everything must be 100%)